data.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts,neutered) VALUES ('Agumon',"Feb 3rd, 2020",10.23,0,1);
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts,neutered) VALUES ('Gabumon',"Nov 15th, 2018",8,2,1);
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts,neutered) VALUES ('Pikachu',"Jan 7th, 2021,",15.04,0);
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts,neutered) VALUES ('Devimon',"Jan 7th, 2021,",15.04,1);
```

Comment:
- [ ] Good job adding the values as requested, however, you need to ensure that your `neutered` values fit the criteria for **`Bit`** data types. Bit data types are **strings** of 1's and 0's, like so: `'1'`, `'0'`.

  ![neutered-bit](https://user-images.githubusercontent.com/36542986/165253609-38d0b11d-6c0d-452c-aca2-de11dc534d95.jpg)

- [ ] **[OPTIONAL]** The previous reviewer probably missed it due to oversight. Your value for `date_of_birth` do not meet the **`Date`** data type. It should be something like so: `03/02/2020`.

  ![wrong-date](https://user-images.githubusercontent.com/36542986/165254628-5b611c3e-3f64-4930-849d-a1ba7113ea03.jpg)